src/components/Worker.tsx

```diff
@@ -0,0 +1,111 @@
+interface WorkerProps {workerInfo: {id: number, firstName: string, lastName:string,
+     title:string, country: string, city:string, birthDate:Date, imageUrl:string, time: string}}
+
+const Worker = ({workerInfo}: WorkerProps) => {
+    const age: number = new Date().getFullYear() - workerInfo.birthDate.getFullYear();
+    console.log(workerInfo.birthDate);
+    return (
+        <>
```

Comment:
you have used only native html tags, which is great but makes it a bit harder. Use MUI (material UI, a library offered by google for their basic elements design). It will make the code much easier to follow and probably look better as well.